_documentation/user/flags/stealth.md

```diff
@@ -5,4 +5,18 @@ type: Good
 summary: Tank is invisible on radar. Shots are still visible. Sneak up behind enemies!
 ---
 
-...
+Stealth flag allows the player to be invisible to other players on the radar.
+
+## Potency
+
+The only way to locate a player with a Stealth flag is to either see them on a screen, see their shots, hear them jump around, or nullify their flag's effect through the use of the [Seer](../seer/)</a> flag.
+
+With the ability of locating and tracking the movement of tanks on radar being really powerful and commonly utilized, especially among the more experienced players, the Stealth flag is considered to be very powerful regardless of the map. However, the more crowded the map is with objects, the higher the potency of the flag is, as it becomes easier for the holder to hide and plan a surprise attack.
+
+As Stealth does not allow the opponent to prepare for a potential interaction due to not being able to even approximately position the holder on the map via radar, it is considered to be *the* most powerful flag in the game by some.
```

Comment:
This subjective comment does not add value, and would require a lot of justification. I would not include it.